aspnetcore/blazor/tooling.md

```diff
@@ -25,12 +25,9 @@ To create a Blazor app on Windows, use the following guidance:
 
 * Install the latest version of [Visual Studio Preview](https://visualstudio.microsoft.com/vs/preview/) with the **ASP.NET and web development** workload.
 
-<!-- UPDATE 8.0 Confirm whether or not the blazorwasm-empty
-     template survives in 8.0+ -->
-
 * Create a new project:
   * For a Blazor Web App experience (*recommended*), choose the **Blazor Web App** template. Select **Next**.
-  * For a Blazor WebAssembly experience, choose the **Blazor WebAssembly App** template, which includes demonstration code and Bootstrap, or the **Blazor WebAssembly App Empty** template without demonstration code and Bootstrap. Select **Next**.
+  * For a Blazor WebAssembly experience, choose the **Blazor WebAssembly Standalone App** template. Select **Next**.
```

Comment:
The main point of these bullets seems to be to indicate the available Blazor templates. How about we just list the templates by name here and have a sentence about what each template is for?

```suggestion
* Create a new project using one of the available Blazor templates:
  * **Blazor Web App** (*recommended*): Creates a Blazor web app that supports both server and client-side rendering with rich interactivity. 
  * **Blazor WebAssembly Standalone App**: Creates a standalone client web app that can be deployed as a static site.
```